deploy/example_catalog/cr-app-gitlab-ee.json

```diff
@@ -0,0 +1,65 @@
+{
+    "apiVersion": "kubedirector.hpe.com/v1beta1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
+        "name" : "gitlab"
+    },
+
+    "spec" : {
+        "systemdRequired": true,
+        "defaultPersistDirs" : ["/home"],
+        "config": {
+            "roleServices": [
+                {
+                    "serviceIDs": [
+                        "ssh",
+                        "http"
+                    ],
+                    "roleID": "gitlab"
+                }
+            ],
+            "selectedRoles": [
+                "gitlab"
+            ]
+        },
+        "label": {
+            "name": "GitLab",
+            "description": "GitLab apps"
+        },
+        "distroID": "bluedata/gitlab",
```

Comment:
Folks should start using some string other than "bluedata" for the vendor part of these distro IDs. Would you like to be the pioneer? :-)

Reply:
Ok, I will change distroID create new pull requests.